src/wasm/wasm-binary.cpp

```diff
@@ -565,6 +565,35 @@ void WasmBinaryWriter::writeInlineString(const char* name) {
   }
 }
 
+inline bool isHexDigit(char ch) {
```

Comment:
these could be `static` (no need for `inline` I don't think)

Reply:
fixed